apps/api/src/app/controllers/content/content.controller.ts

```diff
@@ -98,6 +99,9 @@ export class ContentController {
         if (isNullOrUndefined(kind)) {
             throw new BadRequestException(`You must include the content kind with this request.`);
         }
+        if (formInfo && (formInfo as CreateProse).tags && (formInfo as CreateProse).tags.length > MAX_FANDOM_TAGS) {
```

Comment:
These boolean checks can be simplified:
```suggestion
        if ((formInfo as CreateProse)?.tags?.length > MAX_FANDOM_TAGS ) {
```
I'd also recommend rephrasing the error message, to clarify what the maximum is. Something like `Too many fandom tags. The max is ${MAX_FANDOM_TAGS}`. (note that you have to use backticks to enable string interpolation like that)

Similar for the saveChanges method below.